frontend/src/pages/modelServing/screens/global/ServeModelButton.tsx

```diff
@@ -67,7 +68,15 @@ const ServeModelButton: React.FC = () => {
   if (!project) {
     return (
       <Tooltip data-testid="deploy-model-tooltip" content="To deploy a model, select a project.">
-        {deployButton}
+        <div>{deployButton}</div>
+      </Tooltip>
+    );
+  }
+
+  if (!isNIMAvailable && isKServeNIMEnabled) {
+    return (
+      <Tooltip content="NIM is not available. Contact your administrator.">
+        <div>{deployButton}</div>
```

Comment:
nitpick: adding a `<div>` around the button is not needed
```suggestion
        {deployButton}
      </Tooltip>
    );
  }

  if (!isNIMAvailable && isKServeNIMEnabled) {
    return (
      <Tooltip content="NIM is not available. Contact your administrator.">
        {deployButton}
```

Reply:
Updated